docs/sphinx/source/contributing.rst

```diff
@@ -7,6 +7,10 @@ Encouraging more people to help develop pvlib-python is essential to our
 success. Therefore, we want to make it easy and rewarding for you to
 contribute.
 
+There is a lot of material in this section, aimed at a variety of
+contributors from novice to expert. Don't worry if you don't (yet)
+understand parts of it.
+
 
 Easy ways to contribute
```

Comment:
I think the word "how" is missing on line 18: " Here are a few ideas for you can contribute, ..."